centrallix-sysdoc/OSDriver_Authoring.md

```diff
@@ -115,497 +170,1032 @@ License:  Copyright (C) 2001-2011 LightSys Technology Services. See LICENSE.txt
+### Function: GetQueryCoverageMask()
+```c
+int xxxGetQueryCoverageMask(pObjQuery this);
+```
+**No documentation provided.**
+
+
+### Function: GetQueryIdentityPath()
+```c
+int xxxGetQueryIdentityPath(pObjQuery this, char* pathbuf, int maxlen);
+```
+**No documentation provided.**
 
-### G.  Managing Object Methods
-Objects may optionally have methods associated with them.  Each method is given a unique name within the object, and can take a single string parameter.  Three functions exist for managing methods.
 
-The first two functions, GetFirstMethod and GetNextMethod, work identically to their counterparts dealing with attributes.  The third function, ExecuteMethod, starts a method executing.  This function takes four parameters:  the inf_v pointer, the name of the method, the optional string parameter, and the oxt* pointer.
 
 ## III Reading the Node Object
-The Node object has content which controls what resource(s) this driver will actually access, so it is important for the driver to access the node object's content.  If the driver's node objects are structure files (which is normally the case when dealing with a remote network resource), then the SN module can make opening the node object much more painless.  It also performs caching automatically to improve performance.
+A driver will commonly configure itself by reading text content from its node object file, at the root of its object subtree.  This content may define what resource(s) a driver should provide, how it should access or compute them, and other similar information.  Most drivers use the structure file format for their node objects because SN module makes parsing, reading, and writing these files easier.  It also performs caching automatically to improve performance.
 
-Note that the Node object will technically ALREADY BE OPEN as an object in the objectsystem.  The OSML does that for you.  If your driver will not use the SN/ST modules, then it should read the node object via the normal objRead() function, and write it via objWrite().  Your driver should NEVER objClose() the node object!  The OSML does that for you.
+- 📖 **Note**: The node object will **already be open** as an object in the ObjectSystem: The OSML does this for each driver.  If a driver does not use the SN/ST modules, then it should read and write the node object directly with `objRead()` and `objWrite()`.  A driver should **NEVER** `objClose()` the node object!  The OSML handles that.
 
-An objectsystem driver will commonly configure itself by reading a text file at the root of its object subtree.  There are two main modules available for making this easier.  
+Although using the structure file format may be complex, it allows significant flexibility.  Data is structured in hierarchies where each sub-object can have named attributes as well as sub-objects.  Centrallix is filled with examples of this, including any `.qy`, `.app`, `.cmp`, or `.cluster` file.
```

Comment:
Flexibility, as well as consistency from one type to another, so the user/admin doesn't need to learn a lot of different node object formats.